app/views/scsb_mailer/_message_details.text.erb

```diff
@@ -0,0 +1,16 @@
+<% if @success == true %>
+  <%= "Title: #{@message[:titleIdentifier]}" %>
+  <%= "Barcode: #{@message[:itemBarcode]}" %>
+  <% unless @message[:requestNotes].blank? %>
+    <%= "Notes: #{@message[:requestNotes]}" %>
+  <% end %>
+
+  <%= "Your Library Account: #{link_to 'https://pulsearch.princeton.edu/account', 'https://pulsearch.princeton.edu/account'}" %>
```

Comment:
You can remove the second arg `https://pulsearch.princeton.edu/account` if the url and display text are the same